erpnext/stock/report/stock_ledger/stock_ledger.py

```diff
@@ -88,19 +99,26 @@ def get_items(filters):
 			.format(" and ".join(conditions)), filters)
 	return items
 
-def get_item_details(items, sl_entries):
+def get_item_details(items, sl_entries, include_uom):
 	item_details = {}
 	if not items:
 		items = list(set([d.item_code for d in sl_entries]))
 
 	if not items:
 		return item_details
 
+	cf_field = cf_join = ""
+	if include_uom:
+		cf_field = ", ucd.conversion_factor"
+		cf_join = "left join `tabUOM Conversion Detail` ucd on ucd.parent=item.name and ucd.uom=%(include_uom)s"
+
 	for item in frappe.db.sql("""
-		select name, item_name, description, item_group, brand, stock_uom
-		from `tabItem`
-		where name in ({0})
-		""".format(', '.join(['"' + frappe.db.escape(i,percent=False) + '"' for i in items])), as_dict=1):
+		select item.name, item.item_name, item.description, item.item_group, item.brand, item.stock_uom{cf_field}
```

Comment:
Hey, great PR! Just write the SQL queries this way henceforth: https://github.com/frappe/erpnext/blob/develop/erpnext/stock/report/stock_ledger/stock_ledger.py#L116

This would help standardize the SQL queries and also since we want the SQL queries to be DB agnostic.

Reply:
Alright thanks @codingCoffee, I'll keep it in mind... the PostgreSQL commits were too sudden for me to adapt earlier